crates/iota-data-ingestion-core/src/util.rs

```diff
@@ -13,15 +13,25 @@ use url::Url;
 pub fn create_remote_store_client(
     url: String,
     remote_store_options: Vec<(String, String)>,
-    timeout_secs: u64,
+    request_timeout: u64,
```

Comment:
I personally always add the unit in the variable name if it is unclear.
```suggestion
    request_timeout_s: u64,
```

Reply:
[fixed](https://github.com/iotaledger/iota/pull/5026/commits/1a18b55f5b046c8962e924ebf0e63ca5fa9e25db#diff-952e3e8eba12b463dd465d979982a73e5b753d8818b86db9a496808319756879R16)